### PR TITLE
Add cid

### PR DESCRIPTION
**Pull request by teddylo57**
-Deserialized cid field
-Need to know format for expected Heartbeat response, cid, and uid?

### CODE REVIEW

**Comment by ericliu121187** [comment]
For heartbeat, could we get away with just `cid` for now? We can add `uid` later if needed

File: backend/src/main/java/poker/models/websocket/HeartbeatResposne.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package poker.models.websocket;
+
+import lombok.Data;
+import poker.enums.websocket.MessageType;
+
+@Data
+public class HeartbeatResposne {
```
Comment (ericliu121187) [inline backend/src/main/java/poker/models/websocket/HeartbeatResposne.java:7]: `HeartbeatResponse`?

File: frontend/src/features/session/index.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ import SessionCookiesEnum from './session-cookies-enum';
 import { v4 as uuidv4 } from 'uuid';
 
 const {
+  WS_CONNECT_FAILURE,
```
Comment (ericliu121187) [inline frontend/src/features/session/index.js:15]: 🙏

File: backend/src/main/java/poker/models/websocket/HeartbeatResposne.java (hunk 1 above)
Comment (teddylo57) [reply backend/src/main/java/poker/models/websocket/HeartbeatResposne.java:7]: Yeah, I think to remove all those other fields, I'll create a specific class for heartbeat response with the fields data, cid, type

**Comment by teddylo57** [comment]
How's `{"data":{"success":true},"cid":"2020-04-30T21:09:58.432Z","type":"WS_PONG"}`

**Comment by ericliu121187** [comment]
Yeah I think that works - question - I forgot why we had `"success": true` in responses, when we use `_FAILURE` or is that when the data is empty we include it? Or just in this case?

**Comment by teddylo57** [comment]
Yeah, I don't think there is a need for both - it's whatever the FE rather parse out, either success or _FAILURE/_SUCCESS

File: backend/src/main/java/poker/models/websocket/HeartbeatResposne.java (hunk 1 above)
Comment (teddylo57) [reply backend/src/main/java/poker/models/websocket/HeartbeatResposne.java:7]: Let me know if it's okay to remove {data}, as success is not needed right?

Comment (ericliu121187) [reply backend/src/main/java/poker/models/websocket/HeartbeatResposne.java:7]: Prob okay ya

**Pull request merged**